api/lib/notifications/notifications.js

```diff
@@ -25,6 +25,10 @@ function getMessageTitle(networks, notification) {
       return `You sent ${data.details.amount.amount} ${
         data.details.amount.denom
       }${data.details.amount.amount !== 1 ? 's' : ''} on ${networkTitle}`
+    case eventTypes.TRANSACTION_CLAIM:
+      return `You claimed ${data.details.amounts[0].amount} ${
```

Comment:
mmm in polkadot that will not work, as amounts are stored in rewards array instead of amounts.

Reply:
Cause the different way to claim rewards in substrate, maybe we need to create a new transaction type called receivedRewards, so we can notificate properly to substrate users.

If not we would need to send multiple notification based on one tx (the claim tx itself), i.e: the claimRewards tx in polkadot should trigger a notification for every rewarded address.

Makes sense?

Reply:
I tested this and works since I also added amounts to the claimRewardsDetailsReducer: https://github.com/luniehq/lunie/pull/4908/files#diff-73b5eb017425c5613c422332210b8aca2b8552848feea089a9e474df6140253dR446

You can check in staging DB, the data was properly stored and it follows the same format than in Cosmos, which makes it easier to manage. In any case, this wasn't what was preventing the notifications to get stored, but that TRANSACTION_CLAIM was missing in `notifications.js`

But if I'm missing something here, please let me know. So what you are saying is that not all those rewards go to that address (the iniatiator)? In involvedAddresses we only get one, the one from the initiator. So if I understand you correctly we also need to fix that